src/__tests__/nav.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, fireEvent, render, screen} from '@testing-library/react';
+import Toggle from '../components/CustomInputs/Toggle';
+import Nav from '../components/Nav';
+
+describe('Nav tests', () => {
+  afterEach(cleanup);
+
+  describe('Ensure components get rendered', () => {
```

Comment:
Instead of having lots of shorts tests you may try [with longer](https://kentcdodds.com/blog/write-fewer-longer-tests) tests.